tests/components/MapboxMap.stories.tsx

```diff
@@ -55,3 +60,18 @@ CustomPin.play = async ({ canvasElement }) => {
   userEvent.click(mapPin);
   await canvas.findByText('title1');
 };
+
+export const StaticImageBeforeLoad: Story<MapboxStaticImageProps> = (args) => (
+  <SearchHeadlessContext.Provider value={generateMockedHeadless(locationVerticalSingle)}>
+    <div className='grid h-full w-full'>
+      <MapboxStaticImage {...args} />
+    </div>
+  </SearchHeadlessContext.Provider>
+);
+StaticImageBeforeLoad.args = {
```

Comment:
out of curiosity, since this component has different props/args than `MapboxMap`, does Storybook know to update the props in the controls/docs section for this story to match `MapboxStaticImageProps`? or does it still show the info from `MapboxMapProps`?

Reply:
it shows props from `MapboxMapProps`

Reply:
in that case, what do you think about either separating out this story into another file so the controls match the correct props, or switching off the controls in this particular story for any props that don't overlap between the two interfaces? I'm just imagining it may be confusing if someone tries using the controls for `MapboxMap` props and it has no effect on the `MapboxStaticImage`

Reply:
I do like it as part of the MapboxMap stories since it's only use as an internal component to MapboxMap (we don't export MapboxStaticImage. Showing the controls for MapboxMap make sense to me because that is what user will be expose, and if they provide their own mapbox key and style configuration in the control section, that will be use to construct MapboxStaticImage. If you think that's still confusing, I can disable/hide controls for that particular story.

Reply:
oh ok, so the only controls that are configurable are ones that also affect `MapboxStaticImage`? in that case, I'm fine leaving it as is, especially since we're not exporting `MapboxStaticImage`